src/Skia/Avalonia.Skia/GeometryImpl.cs

```diff
@@ -191,6 +208,10 @@ public void UpdateIfNeeded(SKPath? strokePath, IPen? pen)
                 paint.StrokeCap = cap.ToSKStrokeCap();
                 paint.StrokeJoin = join.ToSKStrokeJoin();
                 paint.StrokeMiter = (float)miterLimit;
+
+                if (DrawingContextHelper.TryCreateDashEffect(pen, out var dashEffect))
```

Comment:
Since it's used for caching, we need to record/compare the dash effect too.

Reply:
It's used in `StrokeContains` which can be called kinda frequently.

Reply:
We previously weren't using the dash effect for `StrokeContains`. Should we conditionally enable it?

Reply:
_Previously_ we've only used stroke thickness. It was updated two weeks ago to use some other pen properties so render bounds are calculated properly.

Reply:
Might be worth to extract the code that creates the Path into a separate method

Reply:
Done